docs/maven-plugin.md

```diff
@@ -188,7 +188,11 @@ The following parameters are available for this extension:
 
 #### `gg.jte.nativeimage.NativeResourcesExtension` { #native-resources-extension }
 
-See details about it in the [native-image documentation](pre-compiling.md#graalvm-native-image-support). This extension requires no parameters.
+!!! info "Version note"
+
+    Available since jte ^^**1.10.0**^^.
```

Comment:
What does `^^` do?

Reply:
Underscore. It will be rendered as this:

<img width="280" alt="Screenshot 2024-04-09 at 4 23 09 PM" src="https://github.com/casid/jte/assets/4576/53ce373d-b85c-4b84-87e1-d1b36095f1fd">

If I remember correctly, I did it this way when migrating the docs to MkDocs.

Reply:
To be precise, it is an `insert` that is styled using an underline:

https://facelessuser.github.io/pymdown-extensions/extensions/caret/?h=ins#insert

Reply:
Thanks!